proto/info/edge_node_cluster.proto

```diff
@@ -0,0 +1,237 @@
+// Copyright(c) 2024 Zededa, Inc.
+// All rights reserved.
+
+syntax = "proto3";
+
+package org.lfedge.eve.info;
+option go_package = "github.com/lf-edge/eve-api/go/info";
+option java_package = "org.lfedge.eve.info";
+
+import "google/protobuf/timestamp.proto";
+import "info/info.proto";
+
+enum KubeNodeStatus {
+  KUBE_NODE_STATUS_UNKNOWN = 0;
+  KUBE_NODE_STATUS_READY = 1;
+  KUBE_NODE_STATUS_NOT_READY = 2;
+  KUBE_NODE_STATUS_SCHEDULING_DISABLED = 3;
+  KUBE_NODE_STATUS_OUT_OF_DISK = 4;
+  KUBE_NODE_STATUS_MEMORY_PRESSURE = 5;
+}
+
+message KubeNodeInfo {
+  // Name of the node
+  string name = 1;
```

Comment:
Is the intent that this name, node_name below, and other names in EVE-OS come from which namespace?

FWIW EVE-OS has a hostname (once onboarded to the controller) which is the device UUID, and VM has a hostname (from DHCP I think) which is the app instance UUID. Will this KubeNodeInfo name be one of those?

And will the pod node_name match?

Reply:
> Is the intent that this name, node_name below, and other names in EVE-OS come from which namespace?

This will be all namespaces.

> FWIW EVE-OS has a hostname (once onboarded to the controller) which is the device UUID, and VM has a hostname (from DHCP I think) which is the app instance UUID. Will this KubeNodeInfo name be one of those?

name would be the node name as shown in the controller to match EdgeNodeInfo DeviceName

> And will the pod node_name match?

Yes